Tangem/Modules/Send/Helpers/SendTransactionSender.swift

```diff
@@ -0,0 +1,62 @@
+//
+//  SendTransactionSender.swift
+//  Tangem
+//
+//  Created by Sergey Balashov on 28.06.2024.
+//  Copyright © 2024 Tangem AG. All rights reserved.
+//
+
+import Foundation
+import BlockchainSdk
+import Combine
+
+protocol SendTransactionSender {
```

Comment:
блин, как избавиться от этого масла-маслянного?.. `TransactionSender` занят в BSDK... КАк-то прям режет слух этот двойной сенд. Пока только такой вариант есть:
```suggestion
protocol SendTransactionDispatcher {
```

Reply:
✅